packages/checkbox/src/index.mts

```diff
@@ -96,9 +96,14 @@ export default createPrompt(
 
     const [active, setActive] = useState(bounds.first);
     const [showHelpTip, setShowHelpTip] = useState(true);
+    const [showValidationMessage, setShowValidationMessage] = useState(false);
 
     useKeypress((key) => {
       if (isEnterKey(key)) {
+        if (items.filter(isChecked).length === 0) {
```

Comment:
We'll want to add a new `config` key (`required?: boolean`) to control this new behaviour. Not everyone will want to force a selection.

Tests that exists today shouldn't need to be modified.